inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/queue/MessageQueueServiceImpl.java

```diff
@@ -0,0 +1,555 @@
+                String errorMessage = String.format("Can not create pulsar topic,adminUrl is empty,cluster:%s",
+                        cluster.getName());
+                LOG.error(errorMessage);
+                return errorMessage;
+            }
+            // check namespace
+            String tenant = extParams.get(KEY_TENANT);
+            String namespace = extParams.get(KEY_NAMESPACE);
+            if (StringUtils.isEmpty(tenant) || StringUtils.isEmpty(namespace)) {
+                return String.format("tenant or namespace is null,tenant:%s,namespace:%s", tenant, namespace);
+            }
+            // create admin
+            String adminUrl = extParams.get(KEY_ADMIN_URL);
+            String authentication = extParams.get(KEY_AUTHENTICATION);
+            PulsarAdminBuilder pulsarAdminBuilder = PulsarAdmin.builder().serviceHttpUrl(adminUrl);
+            if (!StringUtils.isEmpty(authentication)) {
+                pulsarAdminBuilder.authentication(AuthenticationFactory.token(authentication));
+            }
+            admin = pulsarAdminBuilder.build();
+            // get topics
+            Topics topicAdmin = admin.topics();
+            String tenantNamespace = tenant + "/" + namespace;
+            List<String> topics = topicAdmin.getList(tenantNamespace);
+            for (String topic : topics) {
+                try {
+                    topicAdmin.delete(topic);
+                } catch (Exception e) {
+                    String errorMsg = String.format("Can not delete Topic:%s,error:%s",
+                            topic, e.getMessage());
+                    LOG.error(errorMsg, e);
+                    return errorMsg;
+                }
+            }
+            return null;
+        } catch (Exception e) {
+            String errorMsg = String.format("Can not clear pulsar Topic of cluster:%s,error:%s",
+                    cluster.getName(), e.getMessage());
+            LOG.error(errorMsg, e);
+            return errorMsg;
+        } finally {
+            if (admin != null) {
+                admin.close();
+            }
+        }
+    }
+
+    /**
+     * createKafkaTopic
+     */
+    private String clearKafkaTopic(InlongClusterEntity cluster) {
+        AdminClient admin = null;
+        java.lang.reflect.Type type = new TypeToken<HashMap<String, String>>() {
+        }.getType();
+        try {
+            String strExtParams = cluster.getExtParams();
+            Gson gson = new Gson();
+            Map<String, String> extParams = gson.fromJson(strExtParams, type);
+            Properties properties = new Properties();
+            properties.putAll(extParams);
+            // create admin
+            admin = AdminClient.create(properties);
+            // check topic exist
+            Set<String> topics = admin.listTopics().names().get();
+            DeleteTopicsResult result = admin.deleteTopics(topics);
+            result.all().get();
+        } catch (Exception e) {
```

Comment:
Just throw exception directly, framework will process. pls, refer to org.apache.inlong.manager.web.config.ControllerExceptionHandler#handleBusinessExceptionHandler

```suggestion
        } catch (Exception e) {
        ....
        throw new BusinessException(errorMsg);
```

Reply:
change to BusinessException